config/initializers/active_admin.rb

```diff
@@ -1,4 +1,9 @@
+require "vue_component.rb"
+AUTO_BUILD_ELEMENTS=[:admin_component, :template, :slot]
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/SpaceAroundOperators: Surrounding space missing for operator =.